lib/models/lookup.js

```diff
@@ -34,13 +34,17 @@ function LookupModelFactory (Model, assert, Errors, Promise, Constants) {
             }
         },
         findByTerm: function (term) {
-            return this.find({
-                or: [
-                    { node: term },
-                    { macAddress: term },
-                    { ipAddress: term }
-                ]
-            });
+            var query = {}; //empty query will find all entries
```

Comment:
@WangWinson it seems  when input is emtpy, waterline return all is the default behavior, so it's reasonable here